### PR TITLE
feat(collector): apply collector schedule modal

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- apply collector schedule modal
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorScheduleModal.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const props = withDefaults(defineProps<IProps>(), {
+    state: 'set',
+});
+
+const collectorPageStore = useCollectorPageStore();
+const collectorPageState = collectorPageStore.$state;
+
+/* Components */
+const handleCloseModal = () => {
+    collectorPageStore.$patch({
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorScheduleModal.vue:37]: praise: 😎$patch😎

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorScheduleModal.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { useCollectorPageStore } from '@/services/asset-inventory/collector/collector-main/collector-page-store';
+import CollectorScheduleForm
+    from '@/services/asset-inventory/collector/shared/collector-forms/CollectorScheduleForm.vue';
+
+interface IProps {
+    state: string
+}
+
+const props = withDefaults(defineProps<IProps>(), {
+    state: 'set',
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorScheduleModal.vue:29]: suggestion: There's a potential for confusion with other `state` term. How about using `mode` instead? (or else)

**Review by yuda110 — APPROVED** [review]
LGTM!!! Very clean :)

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 3]
```diff
@@ -217,6 +226,12 @@ const state = reactive({
 
 /* Components */
 const handleChangeToggle = () => {};
+const handleClickSchedule = () => {
+    collectorPageStore.$patch({
+        visibleModal: true,
+    });
+    collectorScheduleStore.setOriginCollector(props.item.origin);
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:233]: suggestion: I suggest not to bring the concern of schedule form here. 😄

image

**Review by WANZARGEN — APPROVED** [review]
Cool~ I left some comments but you can apply them all on your next PR :)

**Pull request merged**